tests/test_kubectl.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import pytest
+from pytest_container import DerivedContainer
+from pytest_container.container import ContainerLauncher
+from pytest_container.runtime import OciRuntimeBase
+
+from bci_tester.data import KUBECTL_CONTAINERS
+
+
+@pytest.mark.parametrize("ctr_image", KUBECTL_CONTAINERS)
+def test_kubectl(
+    host,
+    container_runtime: OciRuntimeBase,
+    pytestconfig: pytest.Config,
+    ctr_image: DerivedContainer,
+) -> None:
+    engine = container_runtime.runner_binary
+    cluster_name = os.getenv("K3S_CLUSTER_NAME")
+    control_plane = "k3d-" + cluster_name + "-server-0"
+    host.run_expect([0], f"k3d kubeconfig get {cluster_name} > kubeconfig")
```

Comment:
Please try to make use of something entirely container based without requiring external binaries. We don't have k3d available on all openQA workers and getting it in there will be a major headache for QA. A k3s based container would be much preferred tbh